static/js/table-of-contents.js

```diff
@@ -1,17 +1,23 @@
-window.addEventListener('DOMContentLoaded', () => {
+window.addEventListener("DOMContentLoaded", () => {
   const observer = new IntersectionObserver(entries => {
     entries.forEach(entry => {
-      const id = entry.target.getAttribute('id');
+      const id = entry.target.getAttribute("id")
```

Comment:
I know this is personal preferences, but can we end JS lines with semicolons `;`? I feel it's clearer that way.

Reply:
Sure!